ktor-server/ktor-server-plugins/ktor-server-metrics/jvm/test/io/ktor/server/metrics/dropwizard/DropwizardMetricsTests.kt

```diff
@@ -78,10 +78,12 @@ class DropwizardMetricsTests {
     @Test
     fun `should prefix all metrics with baseName`(): Unit = withTestApplication {
         val prefix = "foo.bar"
-        val registry = application.install(DropwizardMetrics) {
+        var registry: MetricRegistry? = null
+        application.install(DropwizardMetrics) {
             baseName = prefix
             registerJvmMetricSets = false
-        }.registry
+            registry = this.registry
```

Comment:
maybe reverse this statement? initialize the registry outside and set it in config block

Reply:
Will do, thanks